### ISSUE TITLE
parameter problem

### ISSUE THREAD

**Issue by todochenxi** [chasers-of-Qs/OEILP#2]
Hi,
Thanks for your work, I found a parameter problem, in train.py: 
```python
parser.add_argument('--omega', type=float, default=1,...),
```
it should be --omega2

**Comment by todochenxi** [comment]
And in trainer.py line 133 
```python
loss = self.params.omega * self.criterion2(score_onto_po ...
```
you used omega

**Comment by chasers-of-Qs** [comment]
Thanks for your issue.
Actually, we used omega and omega2 as alpha and omega in the experimental phase. However, we missed modifying two parameters in trainer.py at release time:
line 83
`loss = loss + self.params.omega2 * loss_type`
line 133
`loss = self.params.omega * self.criterion2(score_onto_po ...`
Which should be:
`loss = loss + self.params.omega * loss_type`
`loss = self.params.alpha * self.criterion2(score_onto_po ...`
Now we have fixed this problem and released the correct code.
Thanks again for your help!

**Comment by todochenxi** [comment]
My pleasure
